src/services/car/__tests__/leasing.test.ts

```diff
@@ -22,4 +22,117 @@ describe("CAR service", () => {
       ).toEqual(null)
     })
   })
+
+  describe("sendLeasingInterest", () => {
+    const leasingInterest = (attributes = {}) => ({
+      name: "Test",
+      phone: "+41781234567",
+      email: "test@test.com",
+      message: "This is a message of a interested customer",
+      ...attributes,
+    })
+
+    describe("valid parameters", () => {
+      beforeEach(() => {
+        fetchMock.mockResponse("")
+      })
+
+      describe("validate only", () => {
+        it("calls validation endpoint", async () => {
+          await sendLeasingInterest(12345, leasingInterest(), {
+            validateOnly: true,
+          })
+
+          expect(fetch).toHaveBeenCalledWith(
+            expect.stringMatching(
```

Comment:
Whats the reason to use a regex over the exact path as string?

Reply:
`fetch` takes a full URL as an argument. I'd rather not parse it but just match the path